viite-UI/src/view/LinkPropertyForm.js

```diff
@@ -450,14 +450,17 @@
           '<p class="center"><a id="error-list-link" class="floating-stops" href="#work-list/roadAddressErrors">TIEOSOITEVERKON VIRHEET</a></p>' +
           '<p class="form form-horizontal"></p>' +
           '<div class="form-initial-state" id="emptyFormDiv">' +
-          '<span class="header-noposition">Aloita valitsemalla projekti.</span>' +
           '<button id="formProjectButton" class="action-mode-btn btn btn-block btn-primary">Tieosoiteprojektit</button>' +
           '<button id="formNameToolButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Tiennimen ylläpito</button>' +
           '<button id="formNodesAndJunctionsButton" class="open-tool-mode-btn btn btn-block btn-primary" style="margin-top: 5px;">Solmut ja liittymät</button>' +
           '</div>';
         rootElement.append(emptyFormDiv);
         $('[id=formProjectButton]').click(function () {
-          $('[id=projectListButton]').click();
+          if (applicationModel.isProjectOpen()) {
+            new ModalConfirm("Projektin muokkaus on kesken. Tallenna muutokset ja/tai poistu Peruuta-painikkeella.");
+          } else {
+            projectListModel.show();
+          }
```

Comment:
I believe we could change this to:
```
projectListModel.show();
```

Since it won't be possible anymore to see this button with a project open.
This code was used for the other button, because we could open another project with another project still open.

Or, we can leave it like this just for safety, although in normal situation `applicationModel.isProjectOpen()` will never return **true**.